CloudKitty/api/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.7-slim-buster
+
+RUN  apt-get update \
+    && apt-get install --no-install-recommends -y \
+    apache2 \
+    libapache2-mod-wsgi-py3 \
+    gcc \
+    python3-dev \
+    && apt-get clean -y \
+    && rm -rf /var/cache/apt \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install cloudkitty
+
+RUN apt-get purge -y \
+    gcc \
+    python3-dev
+
+RUN  useradd cloudkitty --create-home --home-dir /home/cloudkitty \
+    && mkdir -p /var/www/cloudkitty \
+    && cp /usr/local/lib/python3.7/site-packages/cloudkitty/api/app.wsgi /var/www/cloudkitty \
+    && chown -R cloudkitty:cloudkitty /var/www/cloudkitty \
+    && ln -sf /dev/stdout /var/log/apache2/cloudkitty_error.log \
+    && ln -sf /dev/stdout /var/log/apache2/cloudkitty_access.log \
+    && ln -sf /dev/stdout /var/log/apache2/error.log \
+    && ln -sf /dev/stdout /var/log/apache2/access.log
+
+COPY apache_cloudkitty.conf /etc/apache2/sites-available/cloudkitty.conf
+
+RUN  a2ensite cloudkitty
```

Comment:
Hmm maybe this could be done by directly copying the cloudkitty.conf file to `sites-enabled` ?